splash/qtrender_lua.py

```diff
@@ -260,6 +260,7 @@ class Splash(object):
     _result_content_type = None
     _attribute_whitelist = ['commands', 'args', 'tmp_storage',
                             'lua_properties']
+    _result_headers = []
```

Comment:
It should be set in `__init__` - otherwise you'll be updating a class variable shared by all Splash instances.